### DISCUSSION TITLE
How can I Generate load on Canary service? could you please provide detailed information with code if possible.

### DISCUSSION THREAD

**Discussion by nilampatel-engineer** [argoproj/argo-rollouts#1954] [Q&A]
Would you be able to help pls

**Comment by kostis-codefresh** [comment]
Hello

To create traffic you can use any existing tools you are familiar with. Examples

 * [Apache benchmark](https://httpd.apache.org/docs/current/programs/ab.html)
 *  [Siege](https://github.com/JoeDog/siege)
 * [Fortio](https://github.com/fortio/fortio)
 * [Hey](https://github.com/rakyll/hey)

To run it on the canary you can either do it manually from your CI system or from Argo Rollouts itself with https://argoproj.github.io/argo-rollouts/analysis/job/
